test/peppermint_patty/file_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule PeppermintPatty.FileTest do
+  use ExUnit.Case, async: false
+
+  alias PeppermintPatty.File, as: PattyFile
```

Comment:
I see you're always aliasing it to `PattyFile`. Is there any reason to not name the module/file directly like that?

Reply:
(Nit, btw)

Reply:
without context of an alias it kind of sounds like pedophile :| So that is why I kept File as the original name.